routes/users.js

```diff
@@ -2,11 +2,27 @@ const express = require('express');
 const router = express.Router();
 const { getUsers } = require('../utils/storage');
 const { getUsersDonations } = require('../businessRules/donations');
-const { registerUser, updateUser } = require('../businessRules/users');
+const {
+    registerUser,
+    updateUser,
+    checkIfAdmin,
+} = require('../businessRules/users');
 const { deleteUser } = require('../models/users');
 
 router.get('/', (req, res) => {
-    res.status(200).json(getUsers());
+    if (req.body.userLoginSession) {
+        if (checkIfAdmin(req.body.userLoginSession)) {
+            res.status(200).json(getUsers());
+        } else {
+            res.status(403).send(
+                'Access Forbidden. Only Admin has the right to see the users list.'
+            );
+        }
+    } else {
+        res.status(403).send(
+            'Access Forbidden. Only Admin has the right to see the users list.'
+        );
+    }
```

Comment:
This is repeated twice:
```js
res.status(403).send(
    'Access Forbidden. Only Admin has the right to see the users list.'
);
```

Can you simplify the code above to not repeat it?

Reply:
Phind fixed it.